fdbclient/ManagementAPI.actor.cpp

```diff
@@ -55,6 +57,73 @@ bool isInteger(const std::string& s) {
 	return (*p == 0);
 }
 
+// Check if the specified params supported for the given storage type.
+// It will return false if invalid parameters are given.
```

Comment:
How are you handling parameters in the database configuration which are not supported?  Remember in the downgrade case, or even upgrade case, you can change FDB to a different release so that now some parameter is no longer supported.

Reply:
The function is complicated enough that it needs a comment describing in detail what it does.